SpatialGDK/Source/SpatialGDK/Public/Tests/SpatialView/ExpectedViewDelta.h

```diff
@@ -28,7 +29,7 @@ class ExpectedViewDelta
 	ExpectedViewDelta& AddAuthorityLostTemporarily(const Worker_EntityId EntityId, const Worker_ComponentId ComponentId);
 	ExpectedViewDelta& AddDisconnect(const uint8 StatusCode, FString StatusMessage);
 
-	// Compares the Connection Status and the stored Entity Deltas
```

Comment:
oh, it seems like I somehow managed to remove the bits where we tested the connection status before merging it in. That definitely should compare the connection status. we could either do it in this PR or I can add it back in a separate PR

Reply:
Hmm, I modified this further in https://github.com/spatialos/UnrealGDK/pull/2529 where you can also use it to compare subview deltas. Might be worth doing it in a follow up after that one? I can raise a ticket